rules/Renaming/ValueObject/MethodCallRenameWithArrayKey.php

```diff
@@ -1,51 +0,0 @@
-<?php
```

Comment:
@TomasVotruba Ref https://github.com/rectorphp/rector-symfony/pull/276#issuecomment-1320864551, this make rector-symfony error `class MethodCallRenameWithArrayKey` not found